Library/Formula/speedread.rb

```diff
@@ -0,0 +1,19 @@
+class Speedread < Formula
+  desc "Simple terminal-based rapid serial visual presentation (RSVP) reader"
+  homepage "https://github.com/pasky/speedread"
+  url "https://github.com/pasky/speedread/archive/v1.0.tar.gz"
+  version "1.0"
```

Comment:
Try adding a newline at the end of the test string: `"This is a test\n"`. Maybe `speedread` waits for a full line, and the original test did have a newline (`echo` adds a newline by default).